reana_workflow_controller/config.py

```diff
@@ -25,7 +26,12 @@
 WORKFLOW_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
 """Time format for workflow starting time, created time etc."""
 
-PROGRESS_STATUSES = ["running", "finished", "failed", "total"]
+PROGRESS_STATUSES = [
+    ("running", JobStatus.running),
+    ("finished", JobStatus.finished),
+    ("failed", JobStatus.failed),
+    ("total", None),
+]
```

Comment:
Minor: Is there any reason why we opted for `tuple` list instead of `dict` of `PROGRESS_STATUSES`?

Reply:
This variable has been a workaround from the beginning, the solution would be to go for https://github.com/reanahub/reana-workflow-controller/issues/204#issuecomment-829033572. Once the described change would be done, this mapping would disappear, so no need to make it complicated.